src/main/java/models/GameEngine.java

```diff
@@ -12,7 +12,7 @@ public GameEngine(){
         playing = true;
         deck = new Deck();
     }
-
+    int roundCounter = 1;
```

Comment:
Du kan gå in här på github och ta bort denna raden bara, så slipper du göra det i intelliJ